lite2/client.go

```diff
@@ -561,15 +561,32 @@ func (c *Client) verifyHeader(newHeader *types.SignedHeader, newVals *types.Vali
 			panic(fmt.Sprintf("Unknown verification mode: %b", c.verificationMode))
 		}
 	} else {
-		// 2) Otherwise, perform backwards verification
-		// Find the closest trusted header after newHeader.Height
-		var closestHeader *types.SignedHeader
-		closestHeader, err = c.trustedStore.SignedHeaderAfter(newHeader.Height)
+		var firstHeaderHeight int64
+		firstHeaderHeight, err = c.FirstTrustedHeight()
 		if err != nil {
-			return errors.Wrapf(err, "can't get signed header after height %d", newHeader.Height)
+			return err
+		}
+		var closestHeader *types.SignedHeader
+		if newHeader.Height < firstHeaderHeight {
+			c.logger.Info("Using backwards")
+			closestHeader, err = c.TrustedHeader(firstHeaderHeight)
```

Comment:
??? why var name is closestHeader

Reply:
That's what the name was beforehand, I didn't bother to change it